apps/backend/src/database/migrations/1677775665682-migration.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1677775665682 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+        DELETE FROM PAYMENT WHERE AMOUNT = 0::numeric;
+    `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Not sure that this is being done properly here... ?

Reply:
do we have to delete? this will make database short of entries than what is in the files.

Reply:
As per discussion with Karim this morning, he suggested that we filter out the zero dollar payments from garms as we should (hopefully) not be getting these in the future (since zero dollar amounts cannot be reconciled as there is not corresponding deposit). 

I have not updated the api spec to validate this as he mentioned we should discuss with the team on whether or not to reject incoming $0.00 payments through the api, or whether we should continue to filter. (I think it would be best to reject a $0.00 rather than filter once we get this data through the api...?)